ALGORITHMS/Dynamic Programming/Fibonacci numbers/FibonacciNumbers.py

```diff
@@ -1,9 +1,9 @@
-class fibo:
-    def fibonacci(self , n):
-        if n <= 1:
-            return n
-        return fibo().fibonacci(n-1) + fibo().fibonacci(n-2)
-
-
-if __name__ == "__main__":
-	print(fibo().fibonacci(int(input("Enter the number: "))))
+class fibo:
```

Comment:
Please add comments to explain your code

Reply:
Will add them to all the files and I will create a text document in each sub-folder explaining the code in brief